swarm.go

```diff
@@ -196,18 +198,12 @@ func NewSwarm(config *api.Config, mockStore *mock.NodeStore) (self *Swarm, err e
 		self.accountingMetrics = protocols.SetupAccountingMetrics(10*time.Second, filepath.Join(config.Path, "metrics.db"))
 	}
 
-	syncing := stream.SyncingAutoSubscribe
-	if !config.SyncEnabled || config.LightNodeEnabled {
-		syncing = stream.SyncingDisabled
-	}
+	//if !config.SyncEnabled || config.LightNodeEnabled {
```

Comment:
You are already unhooking the old `stream` protocol, so please remove this code, not just commented out, and also remove all the code in `network/stream` that is no longer used. No point in keeping it around, specially when it is not hooked up anymore here in `swarm.go`, and we are directly switching to the new implementation.